Source/Scene/Scene.js

```diff
@@ -709,6 +719,37 @@ define([
         this._cameraVR = undefined;
         this._aspectRatioVR = undefined;
 
+        /**
+         * When <code>true</code>, rendering a frame will only occur when needed as determined by changes within the scene.
+         * Enabling improves performance of the application.
+         * To render a new frame explicitly in this mode, use {@link requestRender}.
+         *
+         * @see Scene#maximumRenderTimeChange
+         * @see Scene#requestRender
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.requestRenderMode = defaultValue(options.requestRenderMode, false);
+        this._isRendering = true;
```

Comment:
Is `_isRendering` the best name?  Should this be `_requestRender`, `_renderDirty`, etc.?